src/com/bjorn/MakeChange/IMakeChange.java

```diff
@@ -0,0 +1,12 @@
+package com.bjorn.MakeChange;
+
+import java.util.HashMap;
+
+/**
+ * Created by bjornjohnson on 1/6/16.
+ */
+public interface IMakeChange {
+    void updateCoinAmountHashMap(String cointType, HashMap<String, Integer> hmap);
```

Comment:
And finally, does this method need to be exposed in the public interface? Would I ever want to `updateCoinAmountHashMap` outside the context of calling `makeChange`?